packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -41,17 +44,53 @@ export default class RunPanelStore {
     get is_clear_stat_disabled() {
         return this.is_running || this.has_open_contract || this.root_store.journal.unfiltered_messages.length === 0;
     }
+    @computed
+    get initial_values() {
+        const initial_values = {};
+        initial_values.set_max_losses = this.set_max_losses || '';
+        initial_values.run_limit = this.run_limit;
+        return initial_values;
+    }
+
+    @action.bound
+    setLimitations(key, value) {
+        this[key] = value;
```

Comment:
Could we set these keys in a `limitations` object rather than on the run panel store itself?